testing/src/scenario/context.py

```diff
@@ -365,6 +365,51 @@ def action(
             kwargs["id"] = id
         return _Event(f"{name}_action", action=_Action(name, **kwargs))
 
+    @staticmethod
+    def custom(
+        event: ops.BoundEvent,
+        *args: Any,
+        **kwargs: Any,
+    ):
+        """Event triggered by a charm library.
+
+        For example, suppose that a library uses a ``DatabaseRequirer`` object
+        to emit a ``DatabaseReadyEvent`` event, with charm code::
+
+            class MyCharm(ops.CharmBase):
+                def __init__(self, framework: ops.Framework):
+                    super().__init__(framework)
+                    self.db = DatabaseRequirer(self, 'db-relation')
+                    framework.observe(self.db.on.ready, self._on_db_ready)
+
+        To emit a `DatabaseReadyEvent` event::
```

Comment:
Adding extra backticks:

```suggestion
        To emit a ``DatabaseReadyEvent`` event::
```

Reply:
Bah, too much switching between Markdown and ReST. I ought to have noticed this, though. Thanks!